usaspending_api/disaster/v2/views/agency/loans.py

```diff
@@ -117,7 +117,7 @@ def build_elasticsearch_result(self, response: dict) -> List[dict]:
         return results
 
     def _build_json_result(self, bucket: dict):
-        info = json.loads(bucket.get("key"))
+        info = json.loads(bucket.get("key").encode("unicode_escape"))
```

Comment:
These should probably have a default value if there's even the remotest chance "key" can be None.

`json.loads(bucket.get("key", "").encode("unicode_escape")) or None`

If there's no chance "key" can be None, then probably

`json.loads(bucket["key"].encode("unicode_escape"))`

Reply:
I thought about that, but it appeared the existing code always expects/assumes data. The original code would have run `json.loads(None)` which raises an exception. Does adding .encode() make it more fragile?

Reply:
Apparently not.